ragstack/tests/integration_tests/conftest.py

```diff
@@ -48,25 +46,6 @@ def get_env_var(name: str) -> str:
     return value
 
 
-@pytest.fixture(scope="session", autouse=True)
-def setup_and_teardown():
```

Comment:
it's better to keep this part and make it per-test so every test starts from a clean database

Reply:
There are tests that run in a specific order and use resources created in previous tests. e.g. first ingests docs into a collection, second searches for data in that collection. I think we can ensure that we don't create more than 5 collections per db in this suite.

Reply:
we reuse collections between unit tests ? that doesn't seem a right approach